project-handbook/technical-infrastructure.md

```diff
@@ -48,6 +54,8 @@ When setting up new lists, make sure to have a look at the list configuration of
 
 The configuration of lists and owners is done manually inside the web interface.
 
+There are very few use-cases for which we use the The MailEasy product as part of INWX's hosting services. This doesn't provide mail boxes but only forwards for certain email addresses and certain domains. We only use this for redirections can cannot be handled by our Uberspace account because of user name overlaps or similar reasons.
```

Comment:
?

Reply:
Does it make sense with this edit?

```suggestion
There are very few use-cases for which we use the The MailEasy product as part of INWX's hosting services. This doesn't provide mail boxes but only forwards for certain email addresses and certain domains. We only use this for redirections that cannot be handled by our Uberspace account because of user name overlaps or similar reasons.
```